.github/workflows/generate_api_index.yaml

```diff
@@ -0,0 +1,29 @@
+name: Generate API Index
+on:
+  push:
+    branches: [master]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1.8.0
+      - name: Checkout googleapis (this repository)
+        uses: actions/checkout@v2
+      - name: Checkout index generator
+        uses: actions/checkout@v2
+        with:
+          repository: dwsupplee/googleapis-api-index-generator
```

Comment:
Should this be `googleapis/googleapis-api-index-generator`?

Reply:
Yep! I opened https://github.com/googleapis/googleapis-api-index-generator/pull/5 to address a few minor issues. We can either merge this now to get this action working right away and update this target after, or wait until Jon has a moment to review and merge.

Reply:
Let's give Jon overnight

Reply:
Updated!